frontend/app/services/subscription-service.server.ts

```diff
@@ -0,0 +1,70 @@
+import moize from 'moize';
+import { z } from 'zod';
+
+import { getAuditService } from '~/services/audit-service.server';
+import { getInstrumentationService } from '~/services/instrumentation-service.server';
+import { getEnv } from '~/utils/env.server';
+import { getLogger } from '~/utils/logging.server';
+
+const log = getLogger('subscription-service.server');
+
+/**
+ * Return a singleton instance (by means of memomization) of the letters service.
+ */
+export const getSubscriptionService = moize(createSubscriptionService, { onCacheAdd: () => log.info('Creating new letters service') });
+
+function createSubscriptionService() {
+  // prettier-ignore
+  const { 
+    INTEROP_API_BASE_URI,
+  } = getEnv();
+
+  async function getSubscription(sin: string, alertType: string) {
+    const auditService = getAuditService();
+    const instrumentationService = getInstrumentationService();
+    auditService.audit('alert-subscription.get', { sin });
+
+    const url = new URL(`${INTEROP_API_BASE_URI}/dental-care/applicant-information/dts/v1/subscriptions`);
+    url.searchParams.set('sin', sin);
+    url.searchParams.set('alertType', alertType);
+
+    const response = await fetch(url, {
+      // Using POST instead of GET due to how sin params gets logged with SIN
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    instrumentationService.countHttpStatus('http.client.interop-api.alert-subscription.posts', response.status);
```

Comment:
We currently don't plan on using Interop for fetching/updating subscriptions but rather our own API. Also, we are currently using a `GET`:

```suggestion
    instrumentationService.countHttpStatus('http.client.cdcp-api.subscriptions.gets', response.status);
```